editoast/src/views/mod.rs

```diff
@@ -207,7 +209,7 @@ fn service_router() -> router::DocumentedRouter {
                             })
                     })
             })
-            .route("/similar_trains", post!(timetable::similar_trains::similar_trains)) // TODO: put under /timtable
+            .route("/similar_trains", post!(timetable::similar_trains::similar_trains)) // TODO: put under /timetable
```

Comment:
```suggestion
            .route("/similar_trains", post!(timetable::similar_trains::similar_trains))
```

no need since we don't require a timetable id anymore